includes/polling/wireless/canopy-generic.inc.php

```diff
@@ -290,4 +290,21 @@
         $graphs['canopy_generic_freq'] = true;
         unset($rrd_filename, $freq);
     }
+
+    $downlinkutilization = str_replace('"', "", snmp_get($device, "frUtlLowTotalDownlinkUtilization.0", "-Ovqn", "WHISP-APS-MIB"));
```

Comment:
Do you need the str_replace here?  You should get nicely formatted data here and not need that. (and the line below)